packages/cli/src/index.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
```

Comment:
Since the CLI is usually invoked through yarn, I'm not 100% if we need this, but it's been there forever so let's add it back for now

Reply:
Sorry about this one. For whatever reason I thought I had added this shebang by accident but I'll add it back of course.